GhostNetwork.Profiles.Api/Controllers/ProfilesController.cs

```diff
@@ -18,6 +19,20 @@ public ProfilesController(IProfileService profileService)
             this.profileService = profileService;
         }
 
+        /// <summary>
+        /// Search profiles by ids
+        /// </summary>
+        /// <param name="model">Profile ids</param>
+        /// <response code="200">Returns profile</response>
+        [HttpPost("search")]
+        [ProducesResponseType(StatusCodes.Status200OK)]
+        public async Task<ActionResult<IEnumerable<Profile>>> SearchByIdsAsync([FromBody] ProfilesQueryModel model)
```

Comment:
What is the reason for this endpoint?

Reply:
https://github.com/ghosts-network/profiles/issues/16
Or I'm didn't correct understand task?

Reply:
Got it, my mistake. Forgot about issue